src/lib/converter.ts

```diff
@@ -0,0 +1,434 @@
+        required,
+      } = param
+      const DYNAMIC_REQUEST_VARIABLE =
+        'com.luckymarmot.RequestVariableDynamicValue'
+
+      const variable = request.addVariable(
+        name,
+        parameterExampleValue,
+        description || '',
+      )
+
+      variable.schema = schema as any
+
+      const createDynamicValue = new DynamicValue(DYNAMIC_REQUEST_VARIABLE, {
+        variableUUID: variable.id,
+      })
+
+      variable.required = required || false
+
+      if (param.in === 'path') {
+        envManager.setEnvironmentVariableValue(
+          param.name,
+          parameterExampleValue,
+        )
+        return
+      }
+      if (param.in === 'header') {
+        request.addHeader(
+          name,
+          new DynamicString(createDynamicValue) || parameterExampleValue,
+        )
+        return
+      }
+
+      if (param.in === 'query') {
+        request.addUrlParameter(
+          name,
+          new DynamicString(createDynamicValue) || parameterExampleValue,
+        )
+        return
+      }
+    }
+
+    const parameters = params
+      .filter((param: OpenAPIV3.ParameterObject) => !param.deprecated)
+      .map((parameterItem: OpenAPIV3.ParameterObject) => {
+        const schema = parameterItem.schema
+          ? jsonSchemaParser(parameterItem.schema as OpenAPIV3.SchemaObject)
+          : ''
+        return {
+          ...parameterItem,
+          parameterExampleValue: schema,
+        } as ExtendedParamObject
+      })
+
+    // attach request parameter variables here
+    parameters.forEach(mapParameters)
+    return request
+  }
+
+  private setRequestAuth(
+    securityTypes: OpenAPIV3.SecurityRequirementObject[],
+    request: Paw.Request,
+  ): Paw.Request {
+    const envManager = this.getEnviroment()
+    const document = this.apiParser.api as OpenAPIV3.Document
+    // Because there are no references to refer to
+    if (!document.components || !document.components.securitySchemes)
+      return request
+
+    const securitySchemes = document.components.securitySchemes as {
+      [key: string]: OpenAPIV3.SecuritySchemeObject
+    }
+
+    const security = [...securityTypes]
+      .map(
+        (
+          item: OpenAPIV3.SecurityRequirementObject,
+        ): ExtendedSecuritySchemeObject[] =>
+          Object.keys(item).map((key) => ({
+            ...securitySchemes[key],
+            value: item[key],
+            key,
+          })),
+      )
+      .flat() as ExtendedSecuritySchemeObject[]
+
+    const snakeCase = (str: string): string =>
+      str.replace(/[A-Z]/g, (cap) => `_${cap.toLowerCase()}`)
+
+    function setRequestOAuth2(item: any) {
+      if (!Object.keys(item).includes('flows')) return
+      const grantType = Object.keys(item.flows)[0] as any
+      const authFlow = item.flows[grantType]
+      request.oauth2 = {
+        client_id: '',
+        client_secret: '',
+        authorization_uri: authFlow.authorizationUrl,
+        access_token_uri: authFlow.tokenUrl,
```

Comment:
Finally found why it didn't work, `authFlow.tokenUrl` was `undefined`, therefore `access_token_uri` was also `undefined`, but actually a required param for Paw.

This should solve the problem:

```suggestion
        authorization_uri: authFlow.authorizationUrl || '',
        access_token_uri: authFlow.tokenUrl || '',
```